README.md

```diff
@@ -25,67 +25,106 @@ To install:
 sudo snap install checkbox-edgexfoundry --devmode --edge
 ```
```

Comment:
With the removal of the section header, the section content appear out of context, since only one method is given.